dlt/cli/command_wrappers.py

```diff
@@ -154,10 +156,43 @@ def schema_command_wrapper(file_path: str, format_: str, remove_defaults: bool)
 
 
 @utils.track_command("dashboard", True)
-def dashboard_command_wrapper(pipelines_dir: Optional[str], edit: bool) -> None:
+def dashboard_command_wrapper(
```

Comment:
alternatively do chaning this top level dashboard command, we could also allow the pipeline command to not take a pipeline name but a script file. But that is probably quite confusing.